com.unity.shadergraph/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [7.6.0] - 2021-03-25
 
+### Fixed
```

Comment:
7.6 already released - I'm not sure where to put this change, if we need a new 7.7 section or a 7.6.1. Probably need to ask in gfx-packages.

Reply:
I think that's not correct. Afaik we just release 7.5
If so this is the right place to add

Reply:
we are 7.7 now as we will add support for Scarlet xbox. Master have been bumped, need to merge it, thanks